components/PaymentForm.js

```diff
@@ -49,7 +50,10 @@ function PaymentForm({ onClose = () => null }) {
         return alert(`Amount should be greater than or equal to 1`);
       }
       setIsSubmitting(true);
-      await openRzp(form);
+      await openRzp({
+        ...form,
+        amount: (Number(form.amount) * (1 + PROCESSING_CHARGE_PERCENT / 100)).toFixed(2),
```

Comment:
1. This logic is used at two places so it better to extract into a function and place it in `utils/index.js`
2. Call the extracted function outside so we can use it here and also inside the button text.